### PR TITLE
Implement Movie infoplate + tests

### PR DESCRIPTION
**Pull request by willresen**
**In this PR:**

- Added :hover metadata for Movie components + styling
- Added Enzyme tests for the MovieCarousel and Movie components
- Refactored previous tests to remove unneeded async code
- Added newlines to each EOF

**TODO:**

- Add 'Cast & Crew' + 'Movies featuring...' titles
- Convert CSS to stylized components.
- Migrate assets to S3 (assets currently hosted on a private Vultr server)
- Refactor seeding script + tests for ease of reviewing

### CODE REVIEW

File: spec/react.test.dom.js [hunk 1]
```diff
@@ -32,24 +32,30 @@ describe('App.jsx component', () => {
...
-  test('Clicking a Person component should call fetch()', async (done) => {
-    await wrapper.find('div.headshot').first().simulate('click');
-    setTimeout(() => {
-      wrapper.update();
-      expect(fetch.mock.calls.length).toBe(3);
-      done();
-    }, 0)
+  test('Clicking a Person component should call fetch()', () => {
+    wrapper.find('div.headshot').first().simulate('click');
+    expect(fetch.mock.calls.length).toBe(3);
```
Comment (HBMattoon) [inline spec/react.test.dom.js:37]: just so I'm understanding this test correctly, the '3' specifies the three fields in your DB (_id, name, thumbnail_url) that will be returned when your fetch/get request is made, right?

**Review by HBMattoon — APPROVED** [review]
lgtm - Have you decided to use your own site to host the media files instead of using s3? I noticed you switched some of the icons over to your site.

File: spec/react.test.dom.js (hunk 1 above)
Comment (willresen) [reply spec/react.test.dom.js:37]: @HBMattoon 3 is the number of times the fetch mock is called - first for the test setup, then to request the initial movie (beforeEach), and lastly to retrieve the repertoire for the clicked personnel. In my next PR the simulated 'click' will be scoped to the test.

**Comment by willresen** [comment]
> lgtm - Have you decided to use your own site to host the media files instead of using s3? I noticed you switched some of the icons over to your site.

@HBMattoon for now, yes! I will migrate to AWS when the service is deployed.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: 'hackreactor',
```
Comment (therobinkim) [inline .eslintrc.js:2]: Use the Airbnb config instead.

File: babel.config.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 module.exports = function (api) {
   const isTest = api.env('test');
-  const presets = ['@babel/preset-env', '@babel/preset-react']
+  const presets = ['@babel/preset-env', '@babel/preset-react'];
   const plugins = ['@babel/plugin-transform-runtime'];
```
Comment (therobinkim) [inline babel.config.js:4]: What's the plugin for?

File: client/dist/style.css [hunk 4]
```diff
@@ -2,6 +2,15 @@ body{
   background-color: #041c2c;
 }
 
+.main_title{
```
Comment (therobinkim) [inline client/dist/style.css:5]: You should use CSS Modules or StyledComponents asap.

File: client/src/App.jsx [hunk 5]
```diff
@@ -28,23 +29,23 @@ class App extends React.Component {
...
       method: 'GET',
       headers: { 'Content-Type': 'application/json' }
     })
     .then(results => results.json())
-    .then(results => this.setState({featuredPersonnel: results}))
+    .then(results => this.setState({featuredPersonnel: results, selectedPersonnel: id.name}))
   }
 
   render() {
     return this.state.personnel.length ? (
```
Comment (therobinkim) [inline client/src/App.jsx:41]: I think might be easier to read as:
```
render() {
  if(!this.state.personnel.length) {
    return <div>Loading...</div>;
  }
  return (
    <div className="container">
      ...
    </div>
  );
}
```

File: database/seeder.js [hunk 6]
```diff
@@ -88,6 +88,6 @@ const seed = async () => {
     }
 
   });
-}/**********************************/
+};/**********************************/
```
Comment (therobinkim) [inline database/seeder.js:91]: I think having lines like `/********/` might be a decent indicator that code in that section could be refactored into its own file.

File: babel.config.js (hunk 3 above)
Comment (willresen) [reply babel.config.js:4]: @therobinkim It was suggested in the babel docs ([https://babeljs.io/docs/en/babel-plugin-transform-runtime](url)) to run the plugin alongside `@babel/runtime` for async code - I just removed the line and don't see a noticeable difference. I'll remove the plugin in the next PR - thank you!

File: client/dist/style.css (hunk 4 above)
Comment (willresen) [reply client/dist/style.css:5]: @therobinkim Converting to StyledComponents now!

File: babel.config.js (hunk 3 above)
Comment (therobinkim) [reply babel.config.js:4]: @willresen (FYI that hyperlink didn't actually work for me; I had to copy/paste the text into my URL bar.) `@babel/runtime` seems useful according to the docs (whereas `@babel/plugin-transform-runtime` seems to only affect your dev environment? not sure).. I wouldn't just take it out necessary, but I'd experiment with it to see how your development and production bundle sizes change as the number of files you're putting through Babel increases over time.

Comment (willresen) [reply babel.config.js:4]: @therobinkim My bad for the broken link - not sure what happened there. Great suggestion, I'll toggle it occasionally to check the difference.

Comment (willresen) [reply babel.config.js:4]: @therobinkim Ah! I remember now why I installed it. Without the transformer, I recieve `ReferenceError: regeneratorRuntime is not defined` when running Jest tests on async code.

**Pull request merged**